tests/acceptance/preprocessor-smoke-test-slow.js

```diff
@@ -67,7 +67,7 @@ describe('Acceptance: preprocessor-smoke-test', function() {
       });
   });
 
-  it.only('addon registry entries are added in the proper order', function() {
+  it('addon registry entries are added in the proper order', function() {
```

Comment:
Further investigation this is exiting with 1 because build is looking for a css preprocessor.  Changing `tests/fixtures/preprocessor-tests/app-registry-ordering/app/styles/app.css` to `app.scss` allows the tests to pass.